django-cloudlaunch/baselaunch/view_helpers.py

```diff
@@ -86,6 +87,13 @@ def get_credentials_from_request(cloud, request):
                     }
         else:
             return {}
+    elif isinstance(cloud, models.GCE):
+        gce_credentials_json = request.META.get('HTTP_CL_GCE_CREDENTIALS_JSON')
```

Comment:
I think it makes sense to leave it as a single header, since this would correspond directly to the gce_service_creds_dict on the cloudbridge side. On the UI side, we could have an upload field so that the user can upload his/her credentials file, which would be subsequently sent across through this header if it's being used as temporary creds, or saved in the "credentials" database field, if it's going to be persistent.